src/commonmark/en/content/implementation/support.md

```diff
@@ -103,3 +106,23 @@ information as possible:
   - Problem description: Describe the problem clearly, why you think it
     is a problem and which behavior you would expect from the system.
 
+Your bug report will be investigated by the developer team and be given a status. 
+If valid it will also be assigned to a developer and approver and eventually be
+fixed. Note that bugfixes are incorporated into the trunk and latest
```

Comment:
Two comments from me - @Philip-Larsen-Donnelly can hopefully confirm and add some suggested wording
- We don't use the word "trunk" anymore, we use "master", I believe
- the bugs/features don't really have an "approver", maybe that wording was from before we had testers

Suggested text: 
Your bug report will be investigated by the Testing/QA team and be given a status.  If valid its status will be set to "TO DO" and will be visible for the development team in their planning of milestones and releases. It can then be assigned to a developer and be fixed. Note that bugfixes are incorporated into the master branch and branches of up to the three latest (supported)  DHIS2 releases - so more testing and feedback to the developer teams leads to higher quality of your software.

(Just to not promise that all bugs will be assigned to developers and fixed - because that doesn't happen)

Reply:
Agree with Karoline, though instead of "It can then be assigned to a developer and be fixed", I would say "It can then be prioritised for fixing by the development team".